utils/vs-code-ext.json

```diff
@@ -270,12 +270,12 @@
                     "id": "statement-editor",
                     "subDirectories": [
                         {
-                            "dirName": "Call a Ballerina library function",
+                            "dirName": "Call a Ballerina package function",
                             "level": 3,
                             "position": 1,
                             "isDir": false,
-                            "url": "/learn/vs-code-extension/references/statement-editor/call-a-ballerina-library-function",
-                            "id": "call-a-ballerina-library-function"
+                            "url": "/learn/vs-code-extension/references/statement-editor/call-a-ballerina-package-function",
```

Comment:
If you change the url, you need to rename the relevant file too. Otherwise it'will give 404.

Reply:
File is already renamed: https://github.com/ballerina-platform/ballerina-dev-website/pull/8197/files#diff-6b7034fc6095d1b47db76dd6e157bb30d1a2ceb7876c2a562d9cbce02321879d